src/libs/actions/Task.ts

```diff
@@ -90,8 +90,12 @@ Onyx.connect({
 /**
  * Clears out the task info from the store
  */
-function clearOutTaskInfo() {
-    Onyx.set(ONYXKEYS.TASK, null);
+function clearOutTaskInfo(skipConfirmation = false) {
+    if (skipConfirmation) {
+        Onyx.set(ONYXKEYS.TASK, {skipConfirmation: true});
```

Comment:
Just making sure set is what you want to do and not merge?

Reply:
Why do we leave skipConfirmation though? I am just trying to figure out what's the user case when this happens.

Reply:
Ah, this is called (with the flag) from `clearOutTaskInfoAndNavigate`.
Basically, at the very beginning, we clear the task and start the new task. In this case, for the new task, we will be skipping confirmation, so that's what we set.
I'm leaning towards its own Onyx key for the flag, like we do for Transactions, though, as it is a bit weird to have an empty Task with just this flag set. What do you think?

Reply:
Nah, it's fine, no worries